### DISCUSSION TITLE
Allow “staggered” shipping costs (add Y amount per X items)

### DISCUSSION THREAD

**Discussion by jannisborgers** [craftcms/commerce#2401] [Ideas]
**What are you trying to do?**
A client sells wine by the bottle. They are shipped in boxes of 6, as is usual in Europe (and worldwide?). The client would like to calculate shipping costs based on groups of 12 bottles.

**What's your proposed solution?**
Add a table below the article price table where you can set a number of items (e.G. 12), for which you can add a fixed amount.

<img width="1256" alt="craft-commerce-staggered-shipping-costs" src="https://user-images.githubusercontent.com/9452571/90643154-87562280-e233-11ea-8e07-0907707a1a40.png">

It’s probably more common that each **opened set** of 12 adds that amount, but a dropdown where you can switch to **completed sets** of 12 is probably a niche addition.

**Additional context**
I tried making this possible with what Commerce offers right now (shipping cost per article, or by weight), but the weight of bottles can differ based on bottle form and filling content (both 0,75 liter or 1 liter are commonly sold together), while a box of 6 bottles can hold bottles with differing weights, so weight-based costs aren’t an option.

**Comment by jannisborgers** [comment]
I just realized I can add additional rules and use the min. and max. quantity conditions to manually add groups of 12 items. Right now I added four of these groups, so up to 48 bottles can be sold, but naturally I would have to manually add more if needed. Realistically, a customer would probably contact the winery to request a fixed fee when ordering large quantities, but an automatic solution could still be useful.

**Comment by lukeholder** [comment]
Thanks for the feedback, when we look to add more features to the shipping engine this will be reviewed. Thanks.
